src/components/chat-message.tsx

```diff
@@ -90,11 +93,17 @@ const ChatMessage: React.FC<ChatMessageProps> = ({ message, ref }) => {
                     </AccordionItem>
                 </Accordion>
               </div>
-            ) : (
+            ) : ((message.displayMode === 'jsonAgentResponse' ? (              
+              (!message.finished ? <>Thinking ...</>  : (
+                <Markdown className={styles.markdown} remarkPlugins={[remarkGfm]}>
```

Comment:
More readable: 
  {message.messageAction?.type === 'agentQuestion'
    ? removeCodeBlocks(message.content) + message.messageAction.params.question
    : removeCodeBlocks(message.content)}

Reply:
Yes, but I've added this "Thinking ..." message which is out of the <Markdown/> so I guess it should stay as it is (?)